workitem/workitem_repository_blackbox_test.go

```diff
@@ -322,3 +322,47 @@ func (s *workItemRepoBlackBoxTest) TestLookupIDByNamedSpaceAndNumberNotFound() {
 	require.NotNil(s.T(), err)
 	assert.IsType(s.T(), errors.NotFoundError{}, errs.Cause(err))
 }
+
+func (s *workItemRepoBlackBoxTest) TestLookupIDByNamedSpaceAndNumberStaleSpace() {
+	testFxt := tf.NewTestFixture(s.T(), s.DB, tf.Spaces(1), tf.WorkItems(20, func(testf *tf.TestFixture, idx int) error {
```

Comment:
No need to specify `tf.Spaces(1)`. That is a dependency of `tf.WorkItems` if you look at the documentation.

Reply:
Fixed in https://github.com/fabric8-services/fabric8-wit/pull/1643/commits/5f826703ded88dc2c908170876d5982e276daece